### PR TITLE
Update Remotion completions for 3.0

### PR DESCRIPTION
**Pull request by JonnyBurger**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Update Remotion completions with new commands, flags and Remotion Lambda feature

**What is the current behavior? (You can also link to an open issue here)**

Represents the capability of Remotion 2.0 while 3.0 is out now

**What is the new behavior (if this is a feature change)?**

Represents 3.0 functionality

**Additional info:**

### CODE REVIEW

File: src/remotion.ts [hunk 1]
```diff
@@ -1,184 +1,615 @@
...
+  },
+  {
+    name: "--frames-per-lambda",
+    insertValue: "--frames-per-lambda=",
+    description: "How many frames should be rendered per chunk",
+    args: {
+      name: "framesPerLambda",
+    },
+  },
+];
```
Comment (SeparateRecords) [inline src/remotion.ts:79]: Do these options require an `=` separator? Federico fixed the implementation in the autocomplete engine (and we renamed a property) so now you can omit the `insertValue` and set `requiresSeparator: true`, you'll get this behavior for free.

(same goes for the other options further down in this file)

File: src/remotion.ts [hunk 2]
```diff
@@ -1,184 +1,615 @@
...
+    name: "--props",
+    insertValue: "--props=",
+    description: "Pass input props as filename or as JSON",
+    args: {
+      template: ["filepaths"],
+      suggestions: [
+        {
+          type: "arg",
+          displayName: "[json string]",
         },
```
Comment (SeparateRecords) [inline src/remotion.ts:133]: Can use `insertValue` to automatically put the cursor between two quotes, which is likely what you want if you're writing JSON in a shell
```suggestion
        {
          type: "arg",
          displayName: "[json string]",
          insertValue: "'{cursor}'",
        },
```

File: src/remotion.ts [hunk 3]
```diff
@@ -1,184 +1,615 @@
...
+  },
+  {
+    name: "--quality",
+    insertValue: "--quality=",
+    description: "Quality for rendered frames, JPEG only, 0-100",
+    args: {
+      default: "80",
+      suggestions: [0, 10, 20, 30, 40, 50, 60, 70, 80, 90, 100].map((e) => ({
+        name: e.toString(),
+      })),
```
Comment (SeparateRecords) [inline src/remotion.ts:145]: Clever! 😄

File: src/remotion.ts [hunk 4]
```diff
@@ -1,184 +1,615 @@
...
+                  },
+                },
+                {
+                  name: "--architecture",
+                  description:
+                    "Type of CPU architecture to use for the Lambda function",
+                  insertValue: "--architecture=",
+                  args: {
+                    name: "architecture",
+                    suggestions: [{ name: "x86_64" }, { name: "arm64" }],
```
Comment (SeparateRecords) [inline src/remotion.ts:511]: This isn't documented anywhere but there's actually an icon for CPU architecture! `fig://icon?type=cpu`

**Review by SeparateRecords — CHANGES_REQUESTED** [review]
Looking really good! Not much to change, awesome job on this update to the CLI. Looks like it was all hand written 😮

File: src/remotion.ts (hunk 1 above)
Comment (JonnyBurger) [reply src/remotion.ts:79]: This is nice to hear! I must have been on the old version, yes, I wanted that when you autocomplete this, the options pop up right away.

File: src/remotion.ts (hunk 3 above)
Comment (JonnyBurger) [reply src/remotion.ts:145]: Thanks! 😄 Credits to @MatteoGauthier!

File: src/remotion.ts (hunk 4 above)
Comment (JonnyBurger) [reply src/remotion.ts:492]: Nice, we'll take that!

**Comment by JonnyBurger** [comment]
Thanks a lot for these great improvement suggestions! 🙌 
All fixed up!

**Review by SeparateRecords — APPROVED** [review]

**Comment by SeparateRecords** [comment]
Perfect - awesome job on this (and on 3.0!) 🎉🚀

**Comment by MatteoGauthier** [comment]
Extending fig is so easy, thanks for that

**Pull request merged**